packages/jetpack-redirects/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-redirects",
+	"version": "1.0.0",
+	"description": "Jetpack Redirects helper",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": false,
+	"keywords": [
+		"wordpress",
+		"jetpack"
+	],
+	"author": "Automattic Inc.",
+	"contributors": [
+		"leogermani"
+	],
```

Comment:
I don't mean to take away anyone's credit, but these contributors/authors list usually end up unmaintained and new contributors are not added. That's why we usually just write `author: Automattic` -- this ages well 🙂 

Should be probably removed from `docs/monorepo.md`, too.

Reply:
done in https://github.com/Automattic/wp-calypso/pull/41497/commits/be894bf238c97a2bf61ae7fb90e81fa97c15f914